tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -389,6 +390,37 @@ await Task
                 });
         }
 
+        private static List<string> GetGlobalTags(Tracer tracer, RemoteConfigurationSettings rcmSettings)
```

Comment:
Nit: `GetTags` would be a better name, since this doesn't return _only_ the global tags.

Reply:
Done in 1b25f174b74e2596cff9c6b9bb897894afd526b8.